homeassistant/components/landisgyr_heat_meter/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""The Landis+Gyr Heat Meter integration."""
+from __future__ import annotations
+
+from ultraheat_api import HeatMeterService, UltraheatReader
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_DEVICE, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up heat meter from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Remove this if it's not used.

Reply:
Moved some stuff and I think it's okay now.